packages/system/volumesnapshot-crd-for-tenant-k8s/Makefile

```
@@ -0,0 +1,11 @@
+export NAME=victoria-metrics-operator
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)